index.html

```diff
@@ -297,22 +297,48 @@ <h4 class="connect">LETS CONNECT</h4>
               <span class="skill-name">CSS</span>
             </li>
           </ul>
+        </li>
+      </ul>
 
-          <div class="frameworks-container">
-            <a class="framework--title-link" href="#">
-              <span class="framework--title">Frameworks</span></a
-            >
-            <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
-          </div>
-          <hr />
-          <div class="skills-container">
-            <a class="skills--title-link" href="#">
-              <span class="skills--title">Skills</span></a
-            >
-            <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
+      <div class="frameworks-container">
+        <a class="framework--title-link" href="#">
+          <span class="framework--title">Frameworks</span></a
+        >
+        <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
+      </div>
+      <hr />
+      <div class="skills-container">
+        <a class="skills--title-link" href="#">
+          <span class="skills--title">Skills</span></a
+        >
+        <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
+      </div>
+    </section>
+
+    <section class="contact-me-main-container">
+      <div class="contact-me-section">
+        <h2 class="contact-me-header">Contact me</h2>
+        <p class="contact-me-p">
+          If you have an application you <br>
+          are interested in developing, a <br>
+          feature that you need built or a <br>
+          project that needs coding. I'd <br>
+          love to help with it!
+        </p>
+        <form class="form-div"
+        action="https://formspree.io/f/mlevdeoa"
+        method="POST">
+
+          <input type="text" name="user-name" placeholder="Name" class="form-input-name" maxlength="30" required>
+          <input type="email" name="email" placeholder="Email" class="form-input-email" required> 
+          <textarea name="message"  class="text-message" placeholder="Write your message here..." maxlength="500" required></textarea>
+          <div class="submit-btn-div">
+            <button class="Get-In-Touch-btn" type="submit">
+              Get in touch
+            </button>
```

Comment:
- [ ] I suggest that we stick to the Figma design as much as possible. I suggest that you ensure that the color and background color of the button is changed in accordance the Figma design provided both in different button state.
Figma design 😎 
![image](https://github.com/Nelofarzabi/My_Portfolio/assets/99394374/954c26c1-df6f-496a-afee-bc972b2de16b)